tests/frontend/travis/remote_runner.js

```diff
@@ -67,41 +67,39 @@ var sauceTestWorker = async.queue(function (testSettings, callback) {
   });
 }, 5); //run 5 tests in parrallel
 
-// Firefox
+// 1) Firefox on Linux
 sauceTestWorker.push({
     'platform'       : 'Linux'
   , 'browserName'    : 'firefox'
-  , 'version'        : ''
+  , 'version'        : 'latest'
 });
 
-// Chrome
+// 2) Chrome on Linux
 sauceTestWorker.push({
     'platform'       : 'Linux'
   , 'browserName'    : 'googlechrome'
-  , 'version'        : ''
+  , 'version'        : 'latest'
 });
 
-/*
-// IE 8
+// 3) Safari on OSX 10.15
 sauceTestWorker.push({
-    'platform'       : 'Windows 2003'
-  , 'browserName'    : 'iexplore'
-  , 'version'        : '8'
+    'platform'       : 'OS X 10.15'
+  , 'browserName'    : 'safari'
+  , 'version'        : '13'
 });
-*/
 
-// IE 9
+// 4) IE 10 on Win 8
 sauceTestWorker.push({
-    'platform'       : 'Windows XP'
+    'platform'       : 'Windows 8'
   , 'browserName'    : 'iexplore'
-  , 'version'        : '9'
+  , 'version'        : '10.0'
```

Comment:
Would it work if we put **IE 11** here? I know it took a lot of time to find these combinations.

IE 10 is practically out of support.

Reply:
I intentionally chose IE10 and Edge to get opposite ends of the spectrum from MS.  You could bring another blob in for IE11 should you wish.  We're supposed to have 5 concurrent tests run but afaik it only does 4(saucelabs is weird) so it wouldn't slow things down at all..

Reply:
Fine. Let's keep IE 10.